go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -1228,17 +1228,34 @@ func (tc *Catalog) FlushCollectionCompactionForVersionedCollection(ctx context.C
 		}
```

Comment:
It's weird that these changes are not guarded by the feature flag. I don't understand that. Do you know?